cmd/client/card_form.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"strings"
+	"unicode"
+
+	"github.com/PaBah/GophKeeper/internal/models"
+	"github.com/PaBah/GophKeeper/internal/utils"
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+type CardScreen struct {
+	inputs     []textinput.Model
+	updateID   string
+	createMode bool
+	focused    int
+	title      string
+}
+
+const (
+	cardNumber      = 0
+	expiryDate      = 1
+	cardHolder      = 2
+	cvv             = 3
+	totalCardFields = 4
+)
+
+func NewCardScreen() *CardScreen {
+
+	cardNumber := textinput.New()
+	cardNumber.Placeholder = "1234 5678 9012 345"
+	cardNumber.Width = 200
+	cardNumber.Focus()
+
+	expiryDate := textinput.New()
+	expiryDate.Placeholder = "MM/YY"
+	expiryDate.Width = 200
+
+	cardHolder := textinput.New()
+	cardHolder.Placeholder = "John Doe"
+	cardHolder.Width = 200
+
+	cvv := textinput.New()
+	cvv.Placeholder = "123"
+	cvv.Width = 200
+
+	return &CardScreen{
+		inputs:  []textinput.Model{cardNumber, expiryDate, cardHolder, cvv},
+		focused: 0,
+		title:   "Please, enter your payment card's details",
+	}
+}
+
+func (form *CardScreen) Update(m *Model, msg tea.Msg) (tea.Model, tea.Cmd) {
+	switch msg := msg.(type) {
```

Comment:
Please fix the linter errors
https://github.com/PaBah/GophKeeper/actions/runs/10768363273/job/29857360609?pr=3